tests/test_workers/test_tasks/test_build_merge_index_image.py

```diff
@@ -631,7 +631,7 @@ def test_add_bundles_missing_in_source_none_missing(
             'csvName': 'bundle2-2.0',
         },
     ]
-    mock_gil.side_effect = ['v=4.5', 'v4.7,v4.6', 'v4.5-v4.8', 'v4.5,v4.6,v4.7']
+    mock_gil.side_effect = ['v=4.5', 'v4.8,v4.7', 'v4.5-v4.8', 'v4.5,v4.6,v4.7']
```

Comment:
This one case is an exact example of what was wrong. IIB should have been comparing 4.6 with the ocp_version parameter `4.6`, but the list's first value  `4.7` was used.